apps/site/src/lib/components/forms/shared/BaseForm.tsx

```diff
@@ -16,12 +15,16 @@ const FIELDS_WITH_OTHER = [
 	"experienced_technologies",
 ];
 
+interface BaseFormProps {
+	applicationType: "HACKER" | "MENTOR" | "VOLUNTEER";
```

Comment:
Suggestion: if you make these Titlecase, they will match the expected `user_record.Role` label values, so the value can be used like a Role at `user.py:144` rather than indexing the enum.